client/my-sites/plugins-wpcom/jetpack-plugin-item.jsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import { identity } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import Button from 'components/button';
+import PurchaseButton from './purchase-button';
+
+export const JetpackPluginItem = ( {
+	plugin,
+	siteSlug,
+	isActive,
+	translate = identity
+} ) => {
+	let plan;
+
+	if ( plugin.plan === 'premium' ) {
```

Comment:
I don't think these should be buttons actually. They should be simple `span`s. Looks the same as compact buttons, but functions as jus a label.